RegExpMatch.py

```diff
@@ -0,0 +1,34 @@
+# Regular Expression Matching
+'''Given an input string s and a pattern p, implement regular expression matching with support for '.' and '*' where:
+  '.' Matches any single character.​​​​
+  '*' Matches zero or more of the preceding element.
+  The matching should cover the entire input string (not partial).
+e.g. : 
+    Input: s = "aa", p = "a*"
+    Output: true
+    Explanation: '*' means zero or more of the preceding element, 'a'. Therefore, by repeating 'a' once, it becomes "aa".
+'''
+class Solution:
+    def isMatch(self, s: str, p: str) -> bool:
+        l1=len(s)
```

Comment:
Pls Use Variable Name Which is Readable 
Ex: **inputStringVariable1** Or **inptStrVble1**

Reply:
Ok sir.